src/components/App.jsx

```diff
@@ -7,32 +7,38 @@ import prompts from '../data/prompts.json'
 import History from './History'
 import loadFromLocal from '../lib/loadFromLocal'
 import saveToLocal from '../lib/saveToLocal'
+import WheelComponent from './Wheel'
 
 function App() {
-  const INITIALPROMPT = 'Spin to receive your first prompt.'
-  const LASTPROMPT = `The Wheel is tired.
-  No more spins until you reset.`
+  const INITIAL_PROMPT = 'Spin to receive your first prompt.'
+  const LAST_PROMPT = `That's it. Reset to start again.`
 
   const [currentPrompt, setCurrentPrompt] = useState(
-    loadFromLocal('currentPrompt') ?? INITIALPROMPT
+    loadFromLocal('currentPrompt') ?? INITIAL_PROMPT
   )
   const [history, setHistory] = useState(loadFromLocal('promptHistory') ?? [])
+  const [mustSpin, setMustSpin] = useState(false)
```

Comment:
I find the name of this variable a little bit confusing - does it stand for the state "is spinning"? Maybe "isSpinning" and "setIsSpinning" would be clearer. But this is just my humble opinion.

Reply:
The name really is confusing! But I can't think of one that accurately describes what it does in a short form 😅 As long as `mustSpin === true`, exactly one spin process can be triggered. `mustSpin` needs to become `false` and then revert to `true` for the wheel to spin again.